pkg/util/metric/graphite_exporter.go

```diff
@@ -0,0 +1,85 @@
+package metric
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/util/log"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/graphite"
+	prometheusgo "github.com/prometheus/client_model/go"
+)
+
+// implements Gatherer
+type GraphiteExporter struct {
+	pm  *PrometheusExporter
+	ctx context.Context
+	b   *graphite.Bridge
+}
+
+func MakeGraphiteExporter(pm *PrometheusExporter) GraphiteExporter {
+	return GraphiteExporter{pm: pm}
+}
+
+func (ge *GraphiteExporter) Init(ctx context.Context) error {
```

Comment:
Since this constructs the graphite "Bridge", which has the configured URL and Timeout and such, the GraphiteExporter will also need to respond to cluster settings; you won't be able to depend on setting these variables once at start time.